base/aws/thanos-query.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Do we actually need different variant for this? 

I would rather have people override resources in `kubernetes-manifests` rather than have our own specific resource config here.

base/global should have good defaults and people having bigger load should override once they use this